setup.py

```diff
@@ -40,21 +40,40 @@ def get_requirements(directory):
     ),
 ]
 
+# Ensure the scripts are available in the package directory to be included in the wheel
 os.makedirs('pysynap/scripts', exist_ok=True)
 shutil.copyfile('synap_convert.py', 'pysynap/scripts/synap_convert.py')
+shutil.copyfile('image_from_raw.py', 'pysynap/scripts/image_from_raw.py')
+shutil.copyfile('image_to_raw.py', 'pysynap/scripts/image_to_raw.py')
+shutil.copyfile('image_od.py', 'pysynap/scripts/image_od.py')
 
-# Read the contents of README file
-this_directory = Path(__file__).parent
-long_description = (this_directory / "README.rst").read_text()
+# Create synap_help.py script for handling help command
+with open('pysynap/scripts/synap_help.py', 'w') as f:
+    f.write("""\

```

Comment:
I think yes It's better to have a file rather than to create it dynamically. I also think to create scripts directory containing synap_help.py and can add a symlinks to the original scripts so we don't need to handle this edge case manually.